modules/spring/src/test/java/org/apache/ignite/spring/injection/IgniteSpringBeanSpringResourceInjectionTest.java

```diff
@@ -211,6 +214,18 @@ public void testSpringResourceInjectedInService() throws Exception {
                 /** {@inheritDoc} */
                 @Override Integer getInjectedBean() {
                     Ignite ignite = appCtx.getBean(Ignite.class);
+
+                    try {
+                        assertTrue(GridTestUtils.waitForCondition(() -> {
```

Comment:
Can we just use IgniteServices#serviceProxy() here? As this method waits for the service deployment to complete. See [IGNITE-12894](https://issues.apache.org/jira/browse/IGNITE-12894)

Reply:
Fixed, thanks